c2_utils/src/mfx_defaults.cpp

```diff
@@ -192,7 +192,7 @@ mfxStatus mfx_set_defaults_mfxVideoParam_enc(mfxVideoParam* params)
             break;
         case MFX_CODEC_HEVC:
             params->mfx.CodecProfile = MFX_PROFILE_HEVC_MAIN;
-            params->mfx.CodecLevel = MFX_LEVEL_HEVC_6;
+            params->mfx.CodecLevel = MFX_LEVEL_HEVC_51;
             params->mfx.TargetUsage = MFX_TARGETUSAGE_BEST_SPEED;
```

Comment:
Can you tell the reason for this change?

Thanks,

Reply:
because we set default value to level_5_1 when addparam on reflector.
I think whatever 5_1 or 6, as long as same.

https://github.com/projectceladon/MediaSDK_C2/blob/e51c5d5d73cf0efbea19652f8cb0626e009dab91/c2_components/src/mfx_c2_encoder_component.cpp#L341